ictuwp-plugin-thema-taxonomie.php

```diff
@@ -24,8 +24,11 @@
 }
 
 //========================================================================================================
+// only this plugin should activate the TAX_THEMA taxonomy
+if ( ! taxonomy_exists( TAX_THEMA ) ) {
```

Comment:
Ik probeer te begrijpen waarom dit werkt. Want de _definitie_ van de constant `TAX_THEMA` wordt gedaan _in_ de Class, toch? Dus als die class nooit instantiated is, kan die `TAX_THEMA` niet bekend zijn…

Zou het kunnen dat dit 'werkt' omdat (toevallig) `TAX_THEMA` al gedefinieerd is in de contants.php ofzo?

Ik zou sowieso aanvullen:

```php
if ( ! defined( TAX_THEMA ) or ! taxonomy_exists( TAX_THEMA ) ) { 
```

Reply:
Verder wordt nu _altijd_ de `fn_ictu_thema_load_plugin_textdomain()` uitgevoerd, op regel 102. 
Hier moet, denk ik, ook nog een ifje omheen...